Makefile

```
@@ -0,0 +1,13 @@
+
+# Makefile for building a simple Go application
+
+.PHONY: build test run
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)